SPECS/qemu.spec

```diff
@@ -9,15 +9,15 @@ Summary: qemu-dm device model
 Name: qemu
 Epoch: 2
 Version: 4.2.1
-Release: 4.6.1.1%{?dist}
+Release: 4.6.2.1%{?dist}
 License: GPL
 Requires: xcp-clipboardd
 Requires: xengt-userspace
 ## We broke an interface used by xenopsd-xc without version signalling
 ## so we have to carry a conflicts line to say we broke it.
 Conflicts: xenopsd-xc < 0.123.0
 
-Source0: https://code.citrite.net/rest/archive/latest/projects/XSU/repos/qemu/archive?at=v4.2.1&format=tar.gz&prefix=qemu-4.2.1#/qemu-4.2.1.tar.gz
+Source0: qemu.tar.gz
```

Comment:
I know the version number is not present in the original SRPM, but I'm not sure it's a good idea to modify our spec file using an unknown number version. :thinking: @benjamreis

Reply:
It's happened in the past and usually it's just the name of the tarball changing. We need to make sure the contents hasn't changed. It's an unfortunate consequence of how hotfixes are made at Citrix, as far as I can tell.

But there's more here. The same issue as with kernel hotfixes: the filename is a lie. It's not a compressed tar anymore, it's become an uncompressed tar. When this happens for kernel RPMs, this gives you a 900 MB file. In this case I don't apply the change to the tarball and make sure to keep the previous one instead (and mention it in the commit message of the SRPM import on the XS-x.y branch).

Here's it's "only" 100 MB (instead of 28 MB) but it's not clean. We should probably do the same as I usually do for the kernel.

Reply:
I changed my mind: 100 MB is not so big, and keeping the file from the hotfix will give us less manual work in the future, unless the hotfix generation process is fixed at Citrix.

Just check that the files are actually the same in the archive.